docs/one-click-integrations/jira-integration.md

```diff
@@ -1,29 +1,96 @@
 # Jira integration
 
-Pulse integrates with Jira Cloud to receive data about issues, necessary to calculate the metrics [Lead time and Cycle time](../metrics/lead-cycle-time.md).
+Pulse integrates with Jira Cloud to receive data about:
+
+-   **Issues** - necessary to calculate [Lead time and Cycle time](../metrics/lead-cycle-time.md).
+
+-   **Incidents** - necessary to calculate the following Accelerate metrics:
+
+    -   [Time to recover](../metrics/accelerate.md#time-to-recover)
+
+    -   [Change failure rate](../metrics/accelerate.md#change-failure-rate)
 
 ## Setting up the Jira integration
 
+!!! note
+    -   The Jira integration must be set up by a **Jira administrator**, as Pulse will only have access to Jira resources that the user setting up the integration has access to.
+    -   You can only integrate one Pulse organization with each Jira instance.
+    -   If you want to report incidents using Jira, check the [details of the incident detection strategy](#incident-detection-strategy) before you proceed with the integration setup.
```

Comment:
style: It's a good practice to include a verb in the link text, as it provides a clearer "call to action". Also, perhaps asking the user to **review** the details is more explicit and indicates better what we expect them to do.

```suggestion
    -   If you want to report incidents using Jira, [review the details of the incident detection strategy](#incident-detection-strategy) before you proceed with the integration setup.
```